src/flat_map.rs

```diff
@@ -572,3 +568,83 @@ impl<'a, K, V> ExactSizeIterator for ValuesMut<'a, K, V> {
         self.inner.len()
     }
 }
+
+
+#[cfg(feature = "serde1")]
+mod serde_impl
+{
+    // the serde serialization/deserialization is manually handled to
+    // serialize the FlatMap as a classic map
+    // and not as a vector<K, V>
+    // this way the FlatMap is serialized in json as:
+    // { "k1": "v1", "k2": "v2" }
+    // and not
+    // {"v": [["k1", "v1"],["k2", "v2"]]}
+
+
+    use std::marker::PhantomData;
```

Comment:
Please add an import for `core::marker` in `lib.rs` here: https://github.com/toffaletti/flat_map/blob/a0d4cecf50ffd05260ea0bce6be393e6b979b661/src/lib.rs#L17

That way it can compile on `no_std`